# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatform.h
# Commit: eac17a862c29
# Full Hash: eac17a862c29a52ae1a389c5e55c0a1cec0db7b6
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2020-01-11 09:48:29
# Regressor Bug: 1600472
# File Overlap Count: 2
# Description:
#   Bug 1600472 - Disable allowing sacrificing subpixel anti-aliasing for small screens. r=jrmuizel
#   
#   This patch only allows sacrificing subpixel anti-aliasing when the
#   screen size is larger than WUXGA, and when the force disable pref is not
#   set. In the future, we may also add disable this for high end GPUs.
# ==============================================================================

diff -r b6e5faa0693d -r eac17a862c29 gfx/thebes/gfxPlatform.h
--- a/gfx/thebes/gfxPlatform.h	Fri Jan 10 18:24:13 2020 +0000
+++ b/gfx/thebes/gfxPlatform.h	Fri Jan 10 16:57:55 2020 +0000
@@ -665,6 +665,12 @@
   static void ReInitFrameRate();
 
   /**
+   * Update allow sacrificing subpixel AA quality setting (called after pref
+   * changes).
+   */
+  void UpdateAllowSacrificingSubpixelAA();
+
+  /**
    * Used to test which input types are handled via APZ.
    */
   virtual bool SupportsApzWheelInput() const { return false; }